### PR TITLE
Add String.prototype.normalize

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by JakeChampion** [comment]
This will fail until https://github.com/walling/unorm/pull/44 is merged

**Comment by walling** [comment]
@JakeChampion, it's merged and [unorm 1.5.0 was published](https://www.npmjs.com/package/unorm/v/1.5.0).

**Comment by JakeChampion** [comment]
The tests which are currently failing will start to pass once https://github.com/walling/unorm/pull/46 has been merged and released :-)

**Comment by chee** [comment]
@JakeChampion it looks like that's been merged and released !

File: polyfills/String/prototype/normalize/detect.js [hunk 1]
```diff
@@ -0,0 +1 @@
+'normalize' in String.prototype && "".normalize.length === 0
```
Comment (JakeChampion) [inline polyfills/String/prototype/normalize/detect.js]: Does it need to be a string literal or could we check on the String.prototype directly?
```suggestion
'normalize' in String.prototype && String.prototype.normalize.length === 0
```

File: polyfills/String/prototype/normalize/patch.jsdiff [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Index: polyfill.js
+--- polyfill.js
++++ polyfill.js
+@@ -419,7 +419,6 @@ 
+    */
+     unorm.shimApplied = false;
+ 
+-   if (!String.prototype.normalize) {
```
Comment (JakeChampion) [inline polyfills/String/prototype/normalize/patch.jsdiff]: 👏

**Comment by JakeChampion** [comment]
LGTM @chee

**Comment by chee** [comment]
@JakeChampion this is exactly the same as the `fetch` one, it might be worth making the `patch.jsdiff` a convention, and then we can make that something you can set in the config.json somehow ^_^ :)!

**Pull request merged**